ivy_tests/test_ivy/test_frontends/test_torch/test_utilities.py

```diff
@@ -0,0 +1,42 @@
+# global
+from hypothesis import strategies as st
+
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+@handle_frontend_test(
+    fn_tree="torch.result_type",
+    dtype_x_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("valid"),
+        valid_axis=True,
+        max_axes_size=1,
+    ),
+    keepdims=st.booleans(),
+)
```

Comment:
I don't think that neither `axis` nor `keepdims` are needed here

Reply:
@guillesanbri axis and keepdims are mentined in this 
ivy_tests/test_ivy/test_functional/test_core/test_utility.py

that is the reason i'm using them.